backend/src/main/java/org/globe42/web/tasks/TaskController.java

```diff
@@ -92,10 +92,13 @@ public TaskController(TaskDao taskDao,
     @ResponseStatus(HttpStatus.CREATED)
     public void assign(@PathVariable("taskId") Long taskId, @Validated @RequestBody TaskAssignmentCommandDTO command) {
```

Comment:
I'd rather leave the NotNull annotation in TaskAssignmentCommandDTO there, and use a separate, `DeleteMapping("/{taskId}/assignments")` method to delete the assignment. That keeps validation there, keeps the semantics of the command, and looks more RESTful to me.